scripts/package.ts

```diff
@@ -164,6 +164,11 @@ const compileChangesByTemplate = (changelog: string) => {
               .replace(
                 /^([0-9a-f]{7,}): /,
                 (_, commit: string) => `${commitLink(commit)}: `,
+              )
+              // Auto-link a commit hash suffix to GitHub.
```

Comment:
```suggestion
              // Auto-link a commit hash prefix to GitHub.
```
Or maybe "short commit hash"?

Reply:
Ah 🤦, I was referring to it being suffixed on the end of the line, but I can see how this is confusing. Will fix.